cmd/cored/protected_db.go

```diff
@@ -0,0 +1,14 @@
+//+build protected_db
+
+package main
+
+import (
+	"chain/core/config"
+	"chain/database/pg"
+)
+
+func init() {
+	config.BuildConfig.ProtectedDB = true
+}
+
+func resetInDevIfRequested(_ pg.DB) {}
```

Comment:
nbd, but you can leave out the `_` if none of the parameter names in the list are used. So

```go
func resetInDevIfRequested(pg.DB) {}
```